arlo-client/package.json

```diff
@@ -138,7 +138,7 @@
     "@wdio/local-runner": "^5.14.4",
     "@wdio/spec-reporter": "^5.16.11",
     "babel-jest": "^24.9.0",
-    "chromedriver": "78.0.1",
+    "chromedriver": "80.0.0",
```

Comment:
Related, or just included in this PR?

Reply:
@eventualbuddha Just included in this PR because it's required to pass CI. We need to customize the image I think to pin the chrome version to match chromedriver.